docs/create-a-ui-component.md

```diff
@@ -0,0 +1,280 @@
+  }
+
+  export default MediaCard
+  ```
+
+* `src/web/theme/ui/molecules/MediaCard/index.js`: will only proxy the
+  MediaCard.js file in order to be able to do imports on the folder directly.
+
+  ```jsx
+  import MediaCard from "./MediaCard.js";
+
+  export default MediaCard;
+  ```
+
+* `src/web/theme/ui/molecules/MediaCard/MediaCard.story.js`: will add a story to
+  the [Storybook](https://storybook.js.org/) of your application. This will
+  serve as living documentation and will allow anyone to easily understand what
+  is MediaCard used for and how to use it. Learn more about our Storybook usage
+  in (#TODO-documentation).
+
+  ```jsx
+  import MediaCard from "./MediaCard.js";
+  import { storiesOf } from "@storybook/react";
+
+  storiesOf("molecules.MediaCard", module).add("default", () => {
+    return <MediaCard />;
+  });
+  ```
+
+Once you've added your component, you must restart the styleguide. Once it is up
+and running, you can view your new story in `molecules > MediaCard`.
+
+Now that you've done that, you can edit the MediaCard component, save, and view
+the changes live in your browser.
+
+<figure style="margin: 1em 0; max-width: 50rem;">
+<img src="https://storybook.js.org/static/demo.f13d28a7.gif" alt="Demo of Storybook hot reloading" />
+<figcaption style="margin: 0.5em; text-align: center">Take a look at this magical development playground!</figcaption>
+</figure>
+
+> In order to learn more about storybook, please refer to TODO Storybook
+> documentation
+
+### Step 3: Implement your component
+
+Now that everything is ready to go, you can do the actual work and implement the
+component. In the case of the MediaCard, it would look like this:
+
+```jsx
+// src/web/theme/ui/molecules/MediaCard/MediaCard.js
+import React from "react";
+import PropTypes from "prop-types";
+
+const MediaCard ({media, children}) => (
+  <div className="media-card">
+    <div className="media-card__media">{media}</div>
+    <div className="media-card__content">{children}</div>
+  </div>
+);
+
+MediaCard.propTypes = {
+  media: PropTypes.node.isRequired,
+  children: PropTypes.node.isRequired
+}
+
+export default MediaCard
+```
+
+#### Styling
+
+If you need to style things with CSS, you have to import the CSS file directly
+in your component file.
+
+```jsx
+// src/web/theme/ui/molecules/MediaCard/MediaCard.js
+import "./MediaCard.scss";
+
+// ... with the rest of your code here
+```
+
+```scss
+// src/web/theme/ui/molecules/MediaCard/MediaCard.scss
+.media-card {
+  display: flex;
+}
+.media-card__media {
+  width: 30%;
+  max-width: 10em;
+}
+```
+
+Please note here, that we are using [Sass](https://sass-lang.com/) (hence the
+`.scss` extension). We believe that it is easier for developers new to the React
+Ecosystem to remain with this well-known CSS preprocessor.
+
+Additionnally, in each Sass files, we will be able to use a list of variables
+that will allow you to keep our theme consitent accross the whole theme. These
```

Comment:
It must read « that will allow us » since it is « we will be able ».

+ typo : « consi**s**tent » (and across)

Reply:
"theme" is repeated in this sentence.

What about : "our *styles* consistent across the whole theme" ?